.github/workflows/tests.yml

```diff
@@ -65,6 +66,14 @@ jobs:
       - run: cargo test --features=custom # custom should do nothing here
       - if: ${{ matrix.toolchain == 'nightly' }}
         run: cargo test --benches
+      - if: ${{ matrix.toolchain == 'stable' }}
+        run: cargo clippy --all
```

Comment:
Running Clippy on stable would mean that we can get failing CI on new stable releases. Usually I prefer to fix Clippy version and bump it manually form time to time.

Maybe we should add a single Clippy job which will check a set of targets using `cargo clippy --target`?

Reply:
+1 to using some fixed nightly Clippy and running it on a bunch of targets in a single job

Reply:
Why Nightly?

Reply:
If we want to run clippy on targets which don't have libstd pre built, we will have to use `-Z build_std`